scripts/deploy/gke.sh

```diff
@@ -78,9 +87,34 @@ gcp_zone = "$X1_GCP_ZONE"
 gcp_project = "$X1_GCP_PROJECT_NAME"
 node_version = "$X1_CLUSTER_VERSION"
 machine_type = "$ICL_GCP_MACHINE_TYPE"
+gke_gpu_driver_version = "$GKE_GPU_DRIVER_VERSION"
+gpu_enabled="$GPU_ENABLED"
```

Comment:
For consistency with other lines:
```suggestion
gpu_enabled = "$GPU_ENABLED"
```